pkg/controller/sidecarset/sidecarset_pod_event_handler.go

```diff
@@ -82,6 +99,7 @@ func (p *enqueueRequestForPod) updatePod(q workqueue.RateLimitingInterface, old,
 		if sidecarSet.Spec.UpdateStrategy.Type == appsv1alpha1.NotUpdateSidecarSetStrategyType {
 			continue
 		}
+		sidecarcontrol.UpdateExpectations.ObserveUpdated(sidecarSet.Name, sidecarcontrol.GetSidecarSetRevision(sidecarSet), newPod)
```

Comment:
A little suggestion: cloud you move this Line to Line 99?

Reply:
If I put it here, I can filter out Pods managed by sidecar of type NotUpdate, right?